webrender/src/frame_builder.rs

```diff
@@ -94,7 +94,7 @@ pub struct FrameBuilder {
 
     stacking_context_store: Vec<StackingContext>,
     clip_scroll_group_store: Vec<ClipScrollGroup>,
-    clip_scroll_group_indices: FastHashMap<ClipAndScrollInfo, ClipScrollGroupIndex>,
+    clip_scroll_group_indices: FastHashMap<ClipAndScrollInfo, usize>,
```

Comment:
Any reason to switch to an untyped index here? I guess because the ```ClipScrollGroupIndex``` type has extra information?

Reply:
yes